### PR TITLE
Bootable JAR and JKube clustering application on OpenShift

### PR DESCRIPTION
**Pull request by yersan**
A step-by-step guide about how to build and deploy a Bootable JAR application on OpenShift by using JKube maven plugin

### CODE REVIEW

File: _posts/2021-02-01-Bootable-jar-jkube-clustering-openshift.adoc [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+----
+
+2. Our bootable JAR application is built by using https://docs.wildfly.org/21/Bootable_Guide.html#wildfly_bootable_JAR_development[wildfly-jar-maven-plugin]. This is the default configuration in our _pom.xml_:
++
+----
+<plugin>
+    <groupId>org.wildfly.plugins</groupId>
+    <artifactId>wildfly-jar-maven-plugin</artifactId>
+    <configuration>
+        <feature-pack-location>wildfly@maven(org.jboss.universe:community-universe)#21.0.0.Final</feature-pack-location>
```
Comment (jmesnil) [inline _posts/2021-02-01-Bootable-jar-jkube-clustering-openshift.adoc:38]: Should we update your demo to use WildFly 22?

File: _posts/2021-02-01-Bootable-jar-jkube-clustering-openshift.adoc [hunk 2]
```diff
@@ -0,0 +1,425 @@
...
+    <feature-pack>
+        <groupId>org.wildfly</groupId>
+        <artifactId>wildfly-galleon-pack</artifactId>
+        <version>21.0.0.Final</version>
+    </feature-pack>
+</feature-packs>
+----
+====
+
+3. The layers section specify the https://docs.wildfly.org/22/Bootable_Guide.html#wildfly_layers[Galleon layers] we want for our application:
```
Comment (jmesnil) [inline _posts/2021-02-01-Bootable-jar-jkube-clustering-openshift.adoc:72]: as you point to the WildFly 22 doc :)

File: _posts/2021-02-01-Bootable-jar-jkube-clustering-openshift.adoc (hunk 1 above)
Comment (yersan) [reply _posts/2021-02-01-Bootable-jar-jkube-clustering-openshift.adoc]: Yes, thanks, updated.

**Review by jmesnil — APPROVED** [review]
👍

**Comment by jamezp** [comment]
My apologies for doing this a week late. Please feel free to ping me directly if a blog post needs to be merged :)

**Pull request merged**